server_admin/topics/identity-broker/social/google.adoc

```diff
@@ -24,50 +24,36 @@ Log in to the link:https://console.cloud.google.com/project[Google Developer Con
-
-.APIs
-image:images/google-api-list.png[]
-
-Click the `Enable` button on this page.  You will get a message that you must create the credentials of your project.
-So click the `Go to Credentials` button.
-
-.Go To Credentials
-image:images/google-go-to-credentials.png[]
-
-You will then be brought to the credentials page.
+Then navigate to the `APIs & Services` section in the Google Developer Console. On that screen, navigate to `Credentials` administration.
 
-NOTE:  If you logout in the middle of this, there is a menu in the top left hand corner.  Select `API Manager` and it
-       will bring you to your desired screen.
+When users log into Google from {project_name} they will see a consent screen from Google which will ask the user
+if {project_name} is allowed to view information about their user profile. Thus Google requires some basic information about the product before creating any secrets for it. For a new project, you have first to configure `OAuth consent screen`.
 
-You will then be asked to specify what credentials you need and what type of data you will be accessing.
+For the very basic setup, filling in the Application name is sufficient. You can also set additional details like scopes for Google APIs in this page.
 
-.Add Credentials
-image:images/google-add-credentials.png[]
+.Fill in OAuth consent screen details
+image:images/google-oauth-consent-screen.png[]
 
-Select `Web server` and `User data` and click the `What credentials do I need?` button.
+The next step is to create OAuth client ID and client secret. Back in `Credentials` administration, navigate to `Credentials` tab and select `OAuth client ID` under the `Create credentials` button.
 
-.Create OAuth ID
-image:images/google-create-oauth-id.png[]
+.Create credentials
+image:images/google-create-credentials.png[]
 
-Next you'll need to create an OAuth 2.0 client ID.  Specify the name you want for your client.  You'll also need to
+You will then be brought to the `Create OAuth client ID` page. Select `Web application` as the application type. Specify the name you want for your client.  You'll also need to
 copy and paste the `Redirect URI` from the {project_name} `Add Identity Provider` page into the
-`Authorized redirect URIs` field.  After you do this, click the `Create client ID` button.
+`Authorized redirect URIs` field.  After you do this, click the `Create` button.
 
-When users log into Google from {project_name} they will see a consent screen from Google which will ask the user
-if {project_name} is allowed to view information about their user profile.  The next Google config screen asks
-you for information about this screen.
+.Create OAuth client ID
+image:images/google-create-oauth-id.png[]
 
-Once you click `Done` you will be brought to the `Credentials` page.  Click on your new OAuth 2.0 Client ID to view
+Once you click `Done` you will be brought to the `Credentials` page and the client ID.  Click on your new OAuth 2.0 Client ID to view
```

Comment:
> will be brought to the `Credentials` page and the client ID.

The sentence doesn't make much sense to me. The page seems to be called just `Credentials`, so I'd leave it as it was.

Also:
> Once you click `Done`

There's no `Done` button anymore.